### ISSUE TITLE
Gamma Correction Option

### ISSUE THREAD

**Issue by Finatra** [ikorb/gcvideo#40]
When using the scanline feature, the image gets significantly darker.
Would it be possible to gamma correct according to the scanline intensity?

The brightness/contrast settings are nice, but don't solve this particular issue.

**Comment by ikorb** [comment]
Yes, that's the nature of artificial scanlines - I don't think I've seen a generator yet that offers gamma-correction for the non-darkened parts of the image.

I think it might be possible, although with the limitation that the video signal is just 8 bits, so you might get banding in the dark parts and loss of color resolution (two input colors mapped to the same output color) in the bright parts.

**Comment by Finatra** [comment]
I can't speak for everyone, but personally I would GLADLY sacrifice some color accuracy/banding for this feature. I mainly play on 31kHz CRT Monitors.

For others, since GCVideo supports Chroma interpolation, I wonder if something similar could be done to mitigate Gamma Correction artifacts.

Cheers :)
